starter-code/public/index.html

```diff
@@ -102,15 +102,16 @@
     <script src="/vendor/scripts/page.js"></script>
 
     <!-- Your secret GitHub token file should be added into your .gitignore file, therefore recreated on every developer's environment. -->
-    <script src="/scripts/models/githubToken.js"></script>
+    <script src="/scripts/views/githubToken.js"></script>
 
     <!-- Script files for the Article resource: -->
     <script src="/scripts/models/article.js"></script>
+      <script src="/scripts/models/repo.js"></script>
     <script src="/scripts/views/articleView.js"></script>
     <script src="/scripts/controllers/articleController.js"></script>
 
-    <!-- TODO: Be sure to include any new script files: -->
-  
+    <!-- DONE: Be sure to include any new script files: -->
```

Comment:
You are missing another file. When you try to run this, you will see the error in the console.